src/utils.js

```diff
@@ -0,0 +1,44 @@
+module.exports = {
+  /**
+   * Just like parseInt, but fixed to base-10
+   */
+  Int: function (str) {
```

Comment:
```suggestion
  Int(str) {
```
Looks like this syntax should work on Node 6: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Method_definitions#browser_compatibility

Reply:
Oh neat. That's roughly what I was looking for! I bet that helps with the recursive call in cleanJSON too! Thanks!